google-datacatalog-mysql-connector/README.md

```diff
@@ -127,9 +127,14 @@ export MYSQL2DC_RAW_METADATA_CSV=mysql_raw_csv (If supplied ignores the MYSQL se
 
 ```
 
-## 3. Run entry point
+## 3. Adapt user configurations
 
-### 3.1. Run Python entry point
+Along with default metadata, the connector can ingest optional metadata as well, such as number of
+rows in each table. Please check configuration file ingest_cfg.yaml and adapt according to your needs. 
```

Comment:
Maybe it makes sense to just move the file to ../config folder then and adapt the calls in my code? So that it is easier for users to just use it, without having to create their own config. And if they want to  change it, they can just go to source and make changes in the file and save?.. or it doesn't work that way?

So do I understand correctly, that if we install the package via PyPI, the root folder would be postgresql/ ?

Reply:
When you install via PyPI, it's going to be installed in their python library folder, like:
`env/lib/python3.7/site-packages/google/datacatalog_connectors/rdbms`.

So this directory is not user friendly, they would need to change the config there.

I like the way the file is being loaded using: `os.getcwd()`. So this way it's going to load the file where the connector is being executed. So we just need to instruct users to have the config file at same execution dir. WDYT?

Reply:
Alright, please have a look at the README if it's clear now.

Reply:
LGTM